File: zad2/src/main/java/com/example/zad2/ui/BMR/BMRViewModel.java
package com.example.zad2.ui.BMR;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BMRViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public BMRViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is BMR calc fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: zad4/src/main/java/com/example/zad4/ui/BMR/BMRViewModel.kt
package com.example.zad4.ui.BMR

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel


class BMRViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "This is BMR calc fragment"
    }
}